Client/Application/AppDelegate.swift

```diff
@@ -103,14 +103,15 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         let profile = getProfile(application)
 
         telemetry = TelemetryWrapper(profile: profile)
-        FeatureFlagsManager.shared.initializeDeveloperFeatures(with: profile)
-        FeatureFlagUserPrefsMigrationUtility(with: profile).attemptMigration()
-        ThemeManager.shared.updateProfile(with: profile)
 
         // Start intialzing the Nimbus SDK. This should be done after Glean
         // has been started.
         initializeExperiments()
 
+        FeatureFlagsManager.shared.initializeDeveloperFeatures(with: profile)
+        FeatureFlagUserPrefsMigrationUtility(with: profile).attemptMigration()
+        ThemeManager.shared.updateProfile(with: profile)
+
```

Comment:
Would this still need to be before `initializeExeriments`? I ask because that function calls `Experiments.shared` wherein we find 
```
guard FeatureFlagsManager.shared.isCoreFeatureEnabled(.nimbus) else {
            return NimbusDisabled.shared
}
``` 
and we set the status of Nimbus as a core feature for the build in `FeatureFlagsManager.shared.initializeDeveloperFeatures(with: profile)`. Or perhaps I am misunderstanding order of operations?

Reply:
Are there any cases when we shouldn't enable Nimbus?

It's possible to enable Nimbus without an endpoint; at this point, I'm not sure it makes sense to feature flag an SDK that we've had in production for sometime.

Reply:
> Are there any cases when we shouldn't enable Nimbus?

Not that  I can think of.

> It's possible to enable Nimbus without an endpoint; at this point, I'm not sure it makes sense to feature flag an SDK that we've had in production for sometime.

That's fair. For my refactor, I was trying to do as little change as possible, hence keeping it in. If we're happy to perma enable Nimbus, I can can remove that core feature flag.